Examples/Numbers/Sources/Numbers/ContentView.swift

```diff
@@ -5,9 +5,21 @@ struct ContentView: View {
 
     var body: some View {
         VStack {
-            Button("Add number") { counter += 1 }
+            Button {
+                counter += 1
+            } label: {
+                Text("Add number")
```

Comment:
Although it's useful for the PR, I think we should keep the example app clean and just use the string label.